### PR TITLE
Fix "Call to a member function getFormSections() on bool"

### PR DESCRIPTION
**Pull request by paladox**
Sometimes $formFactory->getForm can return a bool, so handle it correctly.

### CODE REVIEW

File: includes/specials/SpecialManageWiki.php [hunk 1]
```diff
@@ -96,6 +96,10 @@ public function showWikiForm( $wiki, $module, $special ) {
 
 			$this->reusableFormDescriptor( $module, $options );
 		} else {
+			if ( RemoteWiki::newFromName( $wiki ) == null ) {
+				$this->getContext()->getOutput()->addHTML( '<div class="errorbox">' . wfMessage( 'managewiki-missing' )->escaped() . '</div>' );
```
Comment (JohnFLewis) [inline includes/specials/SpecialManageWiki.php]: $out is defined above

File: includes/specials/SpecialManageWikiDefaultPermissions.php [hunk 2]
```diff
@@ -94,10 +94,17 @@ public static function validateNewGroupName( $newGroup, $nullForm ) {
...
 
 	public function buildGroupView( $group ) {
+		global $wgCreateWikiGlobalWiki;
+
 		$out = $this->getOutput();
 
 		$out->addModules( 'ext.createwiki.oouiform' );
 
+		if ( RemoteWiki::newFromName( $wgCreateWikiGlobalWiki ) == null ) {
+			$this->getContext()->getOutput()->addHTML( '<div class="errorbox">' . wfMessage( 'managewiki-missing' )->escaped() . '</div>' );
```
Comment (JohnFLewis) [inline includes/specials/SpecialManageWikiDefaultPermissions.php]: $out is defined above.

File: includes/formFactory/ManageWikiFormFactory.php [hunk 3]
```diff
@@ -28,18 +28,6 @@ public function getForm(
...
-		}
-
-		if ( $remoteWiki == null ) {
-			$context->getOutput()->addHTML( '<div class="errorbox">' . wfMessage( 'managewiki-missing' )->escaped() . '</div>' );
-			return false;
-		}
-
 		$dbw = wfGetDB( DB_MASTER, [], $wgCreateWikiDatabase );
 
 		$ceMW = ManageWiki::checkPermission( $remoteWiki, $context->getUser() );
```
Comment (JohnFLewis) [inline includes/formFactory/ManageWikiFormFactory.php]: $remoteWiki is undefined

File: includes/formFactory/ManageWikiFormFactory.php [hunk 4]
```diff
@@ -28,18 +28,6 @@ public function getForm(
 	) {
 		global $wgCreateWikiDatabase, $wgCreateWikiGlobalWiki;
```
Comment (JohnFLewis) [inline includes/formFactory/ManageWikiFormFactory.php]: $wgCWGW is no longer needed.

File: includes/specials/SpecialManageWikiDefaultPermissions.php (hunk 2 above)
Comment (paladox) [reply includes/specials/SpecialManageWikiDefaultPermissions.php]: Thanks, done!

File: includes/specials/SpecialManageWiki.php (hunk 1 above)
Comment (paladox) [reply includes/specials/SpecialManageWiki.php]: Thanks, done!

File: includes/formFactory/ManageWikiFormFactory.php (hunk 4 above)
Comment (paladox) [reply includes/formFactory/ManageWikiFormFactory.php]: Thanks, done!

File: includes/formFactory/ManageWikiFormFactory.php (hunk 3 above)
Comment (paladox) [reply includes/formFactory/ManageWikiFormFactory.php]: Thanks, done!

**Pull request merged**